media/src/io/ogg_writer/mod.rs

```diff
@@ -127,8 +127,10 @@ impl<W: Write + Seek> OggWriter<W> {
     ) -> Result<()> {
         self.last_payload_size = payload.len();
         self.last_payload = payload.clone();
+        let n_segments = (self.last_payload_size / 255) + 1;
```

Comment:
What if last_payload_size is just a multiplier of 255?

Reply:
Fixed in https://github.com/webrtc-rs/webrtc/pull/499/commits/191a56fc1d0c95642f47cf05dbd3d289e32a0a29